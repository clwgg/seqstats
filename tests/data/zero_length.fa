>empty1

>empty2

>empty3
