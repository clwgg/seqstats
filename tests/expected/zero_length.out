Total n:	0
Total seq:	0 bp
Avg. seq:	0.00 bp
Median seq:	0.00 bp
N 50:		0 bp
Min seq:	0 bp
Max seq:	0 bp
